tests/clitable_test.py

```diff
@@ -23,7 +23,7 @@
 
 import copy
 import os
-import re
+import regex
```

Comment:
A conditional import here would mean only one test fails if this library is not present